src/transformers/tokenization_utils.py

```diff
@@ -2179,6 +2179,9 @@ def decode(
         else:
             return text
 
+    def batch_decode(self, token_ids: List[List[int]], **kwargs) -> List[str]:
```

Comment:
I would just rename `token_ids` to something like `sequences` or similar to avoid any ambiguity with `decode` having the same argument.